### PR TITLE
kythe: Adding GitHub Action for Kythe run.

### PR DESCRIPTION
**Pull request by mithro**
Signed-off-by: Tim 'mithro' Ansell <me@mith.ro>

### CODE REVIEW

File: .github/workflows/verible-ci.yml [hunk 1]
```diff
@@ -53,6 +52,46 @@ jobs:
...
+
+        - name: Install Kythe
+          run: |
+            source ./.github/settings.sh
+            ./.github/workflows/install-kythe.sh
+
+        - name: Run Kythe
+          run: |
+            source ./.github/settings.sh
+            ./.github/workflows/install-kythe.sh
```
Comment (fangism) [inline .github/workflows/verible-ci.yml:89]: DId you mean `run-kythe.sh`?

**Review by fangism — CHANGES_REQUESTED** [review]
Error: https://github.com/google/verible/pull/631/checks?check_run_id=1527795728

File: .github/workflows/verible-ci.yml (hunk 1 above)
Comment (mithro) [reply .github/workflows/verible-ci.yml]: Fixed.

**Comment by mithro** [comment]
It looks like this might have worked! https://github.com/mithro/verible/runs/1528134507?check_suite_focus=true

**Comment by mithro** [comment]
```
With the provided path, there will be 1 file(s) uploaded
Total size of all the files uploaded is 22921336 bytes
Finished uploading artifact artifact. Reported size is 22921336 bytes. There were 0 items that failed to upload
Artifact artifact has been successfully uploaded!
```
But I can't find were the artifact actually ended up?

**Comment by mithro** [comment]
Looks like the lack of names meant that everything ended up in single artifact called artifact -> https://github.com/mithro/verible/actions/runs/412084704

The zip file does seem to contain a kzip file -> v0.0-806-gf27994a_2020-12-10-0319.kzip

**Comment by mithro** [comment]
```
$ du -h v0.0-806-gf27994a_2020-12-10-0319.kzip
23M     v0.0-806-gf27994a_2020-12-10-0319.kzip
$ file v0.0-806-gf27994a_2020-12-10-0319.kzip
v0.0-806-gf27994a_2020-12-10-0319.kzip: Zip archive data, at least v2.0 to extract
```

**Review by fangism — APPROVED** [review]
Awesome, thanks for working on this!  I'll import right away.

**Pull request closed without merging**